### ISSUE TITLE
Kotlin Bootcamp issue: create a package

### ISSUE THREAD

**Issue by Loganras** [google-developer-training/kotlin-bootcamp#86]
**Describe the problem**
I have had a few instances during this tutorial where my version of IntelliJ IDEA (community edition) does not have the options to Create a Class, or a Package, etc, and it's made the tutorial a little less enjoyable since I've had to wonder if what I was doing is correct or not.

**In which step of the codelab can this issue be found?**
Step number 1 in https://codelabs.developers.google.com/codelabs/kotlin-bootcamp-classes/#2.
The tutorial suggests right cliking on src and going to New > Package
But Package is not an option for me.  I see
- File
- Scratch File
- Directory
- HTML File
- Kotlin Script
- Kotlin WorkSheet
- Editor Config FIle
- Resource Bundle

**Steps to reproduce?**
What are the steps to reproduce the problem?  Just follow the tutorial.  The option doesn't exist.

**Versions**
What version of IntelliJ IDEA are you using?

IntelliJ IDEA 2020.2.3 (Community Edition)
Build #IC-202.7660.26, built on October 6, 2020
Runtime version: 11.0.8+10-b944.34 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 980M
Cores: 8
Non-Bundled Plugins: org.jetbrains.kotlin

**Additional information**

**codelab:** kotlin-bootcamp

**Comment by ralther** [comment]
Yes, the instructions need to be updated.  Instead of right clicking on **src**, you should right click on **src->main->kotlin**.
